utilities/transactions/transaction_lock_mgr.h

```diff
@@ -94,7 +97,10 @@ class TransactionLockMgr {
   // Maps from waitee -> number of waiters.
   HashMap<TransactionID, int> rev_wait_txn_map_;
   // Maps from waiter -> waitee.
-  HashMap<TransactionID, autovector<TransactionID>> wait_txn_map_;
+  using TrackedTrxInfo =
```

Comment:
Actually, I've been thinking that the code might be easier to read if we made a struct instead of a tuple.